script/verify-certificate.processor.ts

```diff
@@ -0,0 +1,34 @@
+import { Gender, INSiPerson } from '../src/class/insi-person.class';
+import { IPKCS12Certificate, PKCS12Certificate } from '../src/class/pkcs12certificate.class';
+import { INSiClient } from '../src/insi-client.service';
+import { IDAM } from '../src/models/env';
+
+export class VerifyCertificateProcessor {
+  public static async verifyCertificate (certificate: IPKCS12Certificate, passPhrase: string, endpoint: string, customizedIdam?: string): Promise<void> {
+    console.log('your certificate is:', certificate);
+
+    PKCS12Certificate.validateINSCertificate(certificate);
+    console.log('------------------TEST TO CALL INS SERVER WITH THE CERTIFICATE AND A TEST USER------------------');
```

Comment:
If this is the function that will be called by the server, it should not contain logs. You might have to move code around to have logs only when needed.

Reply:
It is not